sdk/eventhub/Azure.Messaging.EventHubs.Shared/tests/Processor/PartitionLoadBalancerTests.cs

```diff
@@ -0,0 +1,355 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using NUnit.Framework;
+using Azure.Messaging.EventHubs.Tests;
+using System.Collections.Generic;
+
+namespace Azure.Messaging.EventHubs.Processor.Tests
+{
+    /// <summary>
+    ///   The suite of tests for the <see cref="PartitionLoadBalancer" />
+    ///   class.
+    /// </summary>
+    ///
+    [TestFixture]
+    public class PartitionLoadBalancerTests
+    {
+        private const string FullyQualifiedNamespace = "fqns";
+        private const string EventHubName = "name";
+        private const string ConsumerGroup = "consumerGroup";
+        private readonly CancellationTokenSource tokenSource = new CancellationTokenSource();
```

Comment:
nit:  Not a big concern due to the testing context, but the pedantic part of me twitches at having a disposable here that we're not disposing.   Maybe consider using a dedicated one in each test so that it can be wrapped in a `using` scope?

Reply:
I was just being lazy here and wanted to avoid typing 😄 . Will do.